project/plugins.sbt

```diff
@@ -3,5 +3,6 @@ addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.12.0")
 addSbtPlugin("io.spray" % "sbt-revolver" % "0.7.2")
 
 addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.0.0")
+
```

Comment:
This blank line allowed my local IntelliJ to parse this file; it got errors before.